generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -123,7 +123,7 @@ jhipster:
         corePoolSize: 2
         maxPoolSize: 50
         queueCapacity: 10000
-    #cors: #By default CORS are not enabled. Uncomment to enable.
+    #cors: #By default CORS is disabled. Uncomment to enable.
```

Comment:
We could have this all uncommented, and add an "enable" property, like we have already for many properties -> this would be easier to use, and more consistent with the rest of the code. Then, for your check, you just need to look if it's enabled or not.